src/builder/create_embed.rs

```diff
@@ -299,7 +305,7 @@ impl CreateEmbedFooter {
         self
     }
 
-    /// Set the icon URL's value. This only supports HTTP(S).
+    /// Set the icon URL's value.
```

Comment:
I'm almost certainly sure you can use local files here too.

Reply:
Done :+1: